arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -140,32 +141,36 @@ RocksDBEngine::RocksDBEngine(application_features::ApplicationServer* server)
 
 RocksDBEngine::~RocksDBEngine() {
   shutdownRocksDBInstance(); 
+
+  delete _db;
```

Comment:
Always set _db to nullptr even if object is about to go away.  That way you can tell in a core file if the code "got this far".  And protects from future coder doing a copy/paste of the delete to someplace that needs the _db=nullptr;